### PR TITLE
CVSL-1939-CVL Remove references to hard stop

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: server/views/pages/view/cases.njk [hunk 1]
```diff
@@ -190,51 +182,10 @@
...
-                                </button>
-                        </form>
-                    </div>
-                </li>
-            </ul>
-        </nav>
-
-        {% if hasMultipleCaseloadsInNomis %}
-            {{ prisonsListToDisplay() }}
+            {{ casesTable({licences: licences, tabType: '', probationView: probationView}) }}
```
Comment (andrewrlee) [inline server/views/pages/view/cases.njk:185]: do you still need to pass in the hardcoded `isHardStop: false,` ?

Comment (nishanthmahasamudram) [reply server/views/pages/view/cases.njk:185]: No, we are not using it.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**